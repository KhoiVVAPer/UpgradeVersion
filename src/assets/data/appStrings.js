/* eslint-disable */

const APP_STRINGS = {
  product_warning_str: 'Unfortunately, the desired product is not part of our current product range anymore. Accessories, cleaning products and operating instructions are still available.',
  watchAgainTxt: 'Here you can find the introduction and explaination of the different app functions.',
};

export default APP_STRINGS;
