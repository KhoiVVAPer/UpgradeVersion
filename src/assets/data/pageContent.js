/* eslint-disable */
const headlineText = {
  "id": "content-13",
  "type": "content",
  "config": '{"layout":"manualcontent_headlinetext","values":[{"headline":"Headline","text":"<b>Lorem ipsum dolor sit amet</b>, consetetur sadipscing elitr, <a href=\\"https://www.google.com/\\">sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd</a> gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, <i>no sea takimata sanctus</i> est Lorem ipsum dolor sit amet."}]}'
};

const textContent = {
  "id": "content-11",
  "type": "content",
  "config": '{"layout":"manualcontent_text","values":[{"text":"<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>"}]}'
};

const ribbon = {
  "id": "content-14",
  "type": "content",
  "config": '{"layout":"manualcontent_ribbon","values":[{"headline":"Headline","text":"<b>Lorem ipsum dolor sit amet</b>, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, <i>no sea takimata sanctus</i> est Lorem ipsum dolor sit amet."}]}'
};

const imageHeadlineText = {
  "id": "content-15",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetext","values":[{"headline":"Headline","text":"Lorem <b>ipsum</b> dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.","image":"https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg"}]}'
};

const imageHeadlineTextFiftyFifty = {
  "id": "content-16",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextfiftyfifty","values":[{"headline":"Headline","text":"<p>Lorem ipsum dolor sit amet, <b>consetetur sadipscing</b> elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo <ul><li>dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</li> <li>Lorem ipsum dolor sit amet, consetetur sadipscing elitr,</li></ul> sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>","imageRight":"1","image":"https://s1.kaercher-media.com/products/13171300/main/1/d3.jpg"}]}'
};

const fullWidthImage = {
  "id": "content-17",
  "type": "content",
  "config": '{"layout": "manualcontent_image","values": [{"image": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg"}]}'
};

const imageHeadlineTextTwoCol = {
  "id": "content-18",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetexttwocol","values": [{"image_left": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_left": "Headline left","text_left": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_right": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_right": "Headline right","text_right": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>"}]}'
};

const imageHeadlineTextThreeCol = {
  "id": "content-19",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextthreecol","values": [{"image_left": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_left": "Headline left one","text_left": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_middle": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_middle": "Headline middle column","text_middle": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>","image_right": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_right": "Headline right side","text_right": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget bibendum sodales, augue velit cursus nunc,</p>"}]}'
};

const imageHeadlineTextFourCol = {
  "id": "content-167",
  "type": "content",
  "config": '{"layout":"manualcontent_imageheadlinetextfourcol","values": [{"image_first": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_first": "Headline first","text_first": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_second": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_second": "Headline second","text_second": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_third": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_third": "Headline third","text_third": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>","image_fourth": "https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg","headline_fourth": "Headline fourth","text_fourth": "<p>Maecenas nec odio et ante tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis magna.</p>"}]}'
};

const horizontalRule = {
  "id": "content-rule",
  "type": "content",
  "config": '{"layout":"manualcontent_horizoantalrule"}'
};

const anchors = {
  "id": "anchor",
  "type": "content",
  "config": '{"layout":"manualcontent_anchor","values":[{"targetId":"content-13","title":"Headline Text"},{"targetId":"content-15","title":"Image Headline"},{"targetId":"content-16","title":"Left Image Headline"},{"targetId":"content-167","title":"Four Column"},{"targetId":"content-17","title":"Full Image"},{"targetId":"content-18","title":"Two Column"},{"targetId":"content-19","title":"Three Column"},{"targetId":"table","title":"Table"}]}'
}

const table = {
  "id": "table",
  "type": "content",
  "config": '{"layout":"manualcontent_table","values":[[[{"value":"<b>Column1</b>","style":""},{"value":"<b>Column2</b>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}],[{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""},{"value":"<b>Content</b><ul><li>item</li><li>item</li><li>item</li></ul>","style":""}]]]}'
}

const businessUnitEntry = {
  values: [{
    image_left: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    headline_left: '',
    button_link_left: '',
    button_text_left: '',
    image_right: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    headline_right: '',
    button_link_right: '',
    button_text_right: '',
  }]
}

const imageCollection = {
  values: [{
    imageBig: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    buttonlink_imageBig: '1000',
    headline_imageBig: 'Test',
    buttonlink_imageBig: '345345345',
    buttontext_imageBig: 'Test',
    headline_imageSmallTop: 'Test',
    imageSmallTop: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallTop: 'Test',
    headline_imageSmallMiddle: 'Test',
    imageSmallMiddle: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallMiddle: 'Test',
    headline_imageSmallBottom: 'Test',
    imageSmallBottom: 'https://s1.kaercher-media.com/media/image/file/2207/d1/services.jpg',
    link_imageSmallBottom: 'Test',
  }]
}

export {
  headlineText,
  textContent,
  ribbon,
  imageHeadlineText,
  imageHeadlineTextFiftyFifty,
  fullWidthImage,
  imageHeadlineTextTwoCol,
  imageHeadlineTextThreeCol,
  imageHeadlineTextFourCol,
  horizontalRule,
  anchors,
  table,
  businessUnitEntry,
  imageCollection
};
