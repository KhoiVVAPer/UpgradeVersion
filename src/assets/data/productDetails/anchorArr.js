const anchorArr = [
  { name: 'Anchor1' },
  { name: 'Anchor2' },
  { name: 'Anchor3' },
  { name: 'Anchor4' },
];

export default anchorArr;
