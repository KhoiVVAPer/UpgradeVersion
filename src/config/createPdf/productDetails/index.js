import { Platform } from 'react-native';
import FileViewer from 'react-native-file-viewer';
import RNHTMLtoPDF from 'react-native-html-to-pdf';
import * as globals from '../../libs/globals';
import { getBreadcrumbArr, fontToBase64 } from '../pdfHelpers';
import {
  // pageBreak,
  getHeader,
  getFooter,
  getBreadcrumb
} from '../pdfLayouts';
import styles from '../styles';

import getSection1 from './getSection1';
import getEquipment from './getEquipment';
import getIcons from './getIcons';
import featuresAndBenefits from './featuresAndBenefits';
import compatibleMachines from './compatibleMachines';
import applications from './applications';
import detergentWarnings from './detergentWarnings';
import configurableComponents from './configurableComponents';
import cleaningAgents from './cleaningAgents';
import useOrientation from './useOrientation';

const productDetailsPdf = (productData, structureData, toReturnStr) => new Promise(async (resolve) => {
  const translations = globals.GET_APP_DATA('translationsArr');
  const breadcrumbData = await getBreadcrumbArr(productData.id, structureData);
  const fontUrlArr = await fontToBase64();

  let htmlStr = '';
  htmlStr += await getSection1(productData, translations);
  htmlStr += getEquipment(productData, translations);
  htmlStr += await getIcons(productData);
  htmlStr += await featuresAndBenefits(productData, translations);
  htmlStr += compatibleMachines(productData.compatibleProducts, translations);
  htmlStr += applications(productData.applications, translations);
  htmlStr += detergentWarnings(productData.detergentWarnings, translations);
  htmlStr += configurableComponents(productData.configurableComponents, translations);
  htmlStr += cleaningAgents(productData, translations);

  const finalHtmlStr = `${getHeader(styles.productDetails, fontUrlArr)}
  ${getBreadcrumb(breadcrumbData)}
  ${htmlStr}
  ${getFooter()}
  `;
  if (toReturnStr) {
    resolve(finalHtmlStr);
    return finalHtmlStr;
  }
  let options = {
    html: finalHtmlStr,
    fileName: Platform.OS === 'ios' ? `${globals.APP_FOLDER}/productDetails` : 'productDetails',
    directory: 'Documents',
  };

  if (Platform.OS === 'ios') {
    options = {
      ...options,
      padding: 0,
      bgColor: '#fff'
    };
  }

  const file = await RNHTMLtoPDF.convert(options);
  FileViewer.open(file.filePath)
    .then(() => {
      resolve();
    }).catch(() => { });
  return false;
});

export default productDetailsPdf;
