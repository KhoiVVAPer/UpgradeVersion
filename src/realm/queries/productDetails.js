import Config from 'react-native-config';
import { getDb } from '../../config/libs/helpers';
import appConstants from '../../config/libs/constants';

const { schemas } = appConstants;

/**
 * Save product details into database
 * @param {Int} productId is unique id of product
 * @param {Object} listData is data of product
 */
const saveProductDetailsDb = (productId, listData) => {
  getDb(false, productId).then((realm) => {
    realm.write(() => {
      let id = realm.objects(schemas.PRODUCT_DETAILS).max('id');
      id = id ? id + 1 : 1;

      const productDetailsObj = JSON.parse(listData.data);
      realm.create(schemas.PRODUCT_DETAILS, {
        id,
        productId,
        partnumber: String(productDetailsObj.partnumber),
        export: listData.export,
        year: listData.year,
        country: listData.country,
        language: listData.language,
        createdAt: listData.createdAt,
        data: listData.data,
        productName: listData.data ? JSON.parse(listData.data).name : ''
      });
    });
  });
};

/**
 * Get data of product for provided productId
 * @param {Int} productId is unique id of product
 * @return {Object} product data
 */
const getProductDetailsDb = (productId) => new Promise((resolve) => {
  getDb(false, productId).then((realm) => {
    let productListData = realm.objects(schemas.PRODUCT_DETAILS).filtered(`productId=${productId}`);
    productListData = Array.from(productListData);
    if (productListData.length < 1) {
      resolve(false);
      return;
    }

    productListData = { ...productListData[0] };
    resolve(productListData);
  });
});

/**
 * Get data of product for provided order number
 * @param {Int} partnumberFormatted is unique id of product
 * @return {Object} product data
 */
const getProductDetailsByPartNoDb = (partnumber, db) => new Promise((resolve) => {
  getDb(db).then((realm) => {
    let productListData = realm.objects(schemas.PRODUCT_DETAILS).filtered(`partnumber="${partnumber}"`);
    productListData = Array.from(productListData);
    if (productListData.length < 1) {
      resolve(false);
      return;
    }

    productListData = { ...productListData[0] };
    resolve(productListData);
  });
});

/**
 * Get data of product for rootline
 * @param {Int} id id to finde in rootline for products
 * @return {Object} product data
 */
const getProductDetailsByRootline = (id) => new Promise(async (resolve) => {
  const rootlineId = parseInt(id);

  const getData = (db, rootlineDataId) => new Promise(async (resolveData) => {
    getDb(db).then((realm) => {
      let productListData = realm.objects(schemas.PRODUCT_DETAILS).filtered(`data CONTAINS '"id":${rootlineDataId}'`);
      productListData = Array.from(productListData);

      if (productListData.length < 1) {
        resolveData([]);
        return;
      }
      productListData = productListData.map((item) => {
        const itemFormatted = { ...item };
        return JSON.parse(itemFormatted.data);
      });
      resolveData(productListData);
    });
  });

  const hgProducts = await getData(Config.HOME_AND_GARDEN_DATABASE, rootlineId);
  const proProducts = await getData(Config.PROFESSIONAL_DATABASE, rootlineId);
  const products = [...hgProducts, ...proProducts];
  resolve(products);
});

/**
 * Delete product details
 * @param {Int} productId is unique id of product
 * @return {Boolean} if product is deleted
 */
const deleteProductDetailsDb = (productId) => new Promise((resolve) => {
  getDb(false, productId).then((realm) => {
    realm.write(() => {
      const productList = realm.objects(schemas.PRODUCT_DETAILS).filtered(`productId=${productId}`);
      realm.delete(productList);
      resolve(true);
    });
  });
});

export {
  saveProductDetailsDb,
  getProductDetailsDb,
  getProductDetailsByPartNoDb,
  deleteProductDetailsDb,
  getProductDetailsByRootline
};
