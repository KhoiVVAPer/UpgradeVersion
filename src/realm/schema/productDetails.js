import appConstants from '../../config/libs/constants';

const { schemas } = appConstants;

const productDetailsSchema = {
  name: schemas.PRODUCT_DETAILS,
  primaryKey: 'id',
  properties: {
    id: 'int',
    productId: { type: 'int', optional: true },
    partnumber: { type: 'string', optional: true },
    export: { type: 'string', optional: true },
    year: { type: 'int', optional: true },
    country: { type: 'string', optional: true },
    language: { type: 'string', optional: true },
    createdAt: { type: 'string', optional: true },
    data: { type: 'string', optional: true },
    productName: { type: 'string', optional: true },
    businessUnit: { type: 'string', optional: true },
    productType: { type: 'string', optional: true },
  }
};

export default productDetailsSchema;
