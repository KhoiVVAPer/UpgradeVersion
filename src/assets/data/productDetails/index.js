import * as accessoriesData from './accessories';
import applicationsArr from './applicationsArr';
import detergentWarningsArr from './detergentWarningsArr';
import anchorArr from './anchorArr';

export {
  accessoriesData,
  applicationsArr,
  detergentWarningsArr,
  anchorArr
};
