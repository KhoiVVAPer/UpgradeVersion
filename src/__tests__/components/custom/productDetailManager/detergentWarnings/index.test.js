import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import DetergentWarnings from '../../../../../components/custom/productDetailManager/detergentWarnings';
import { detergentWarningsArr } from '../../../../../assets/data/productDetails';

const setUp = (props) => {
  const component = shallow(<DetergentWarnings {...props} />);
  return component;
};

describe('components > productDetailManager > detergentWarnings', () => {
  let component;
  beforeEach(() => {
    component = setUp({ layoutData: detergentWarningsArr });
    return component;
  });

  it('Check snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders header component', () => {
    const input = component.find('SectionHeader').length;
    const output = 1;
    expect(input).toEqual(output);
  });

  it('Renders flatlist component', () => {
    const input = component.find('FlatList').length;
    const output = 1;
    expect(input).toEqual(output);
  });

  it('Check flatlist data prop', () => {
    const input = component.find('FlatList').prop('data');
    const output = detergentWarningsArr;
    expect(input).toEqual(output);
  });

  it('Check flatlist is two columns', () => {
    const input = component.find('FlatList').prop('numColumns');
    const output = 2;
    expect(input).toEqual(output);
  });

  it('Check if flatlist is not scrollable', () => {
    const input = component.find('FlatList').prop('scrollEnabled');
    expect(input).toBeFalsy();
  });

  it('Check if flatlist contains renderItem prop', () => {
    const input = component.find('FlatList').prop('renderItem');
    expect(input).toBeDefined();
  });

  it('Check if component contains renderItem function', () => {
    const componentInstance = component.instance();
    const input = componentInstance.renderItem;
    expect(input).toBeDefined();
  });

  it('Check if component contains renderItem view is rendered', () => {
    const listWrapper = component.find({ dcaTest: 'listItem' });
    expect(listWrapper).toBeDefined();
  });
});
