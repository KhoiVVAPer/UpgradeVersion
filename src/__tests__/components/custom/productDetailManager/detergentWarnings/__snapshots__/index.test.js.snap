// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > productDetailManager > detergentWarnings Check snapshot 1`] = `
<react-native-mock>
  <react-native-mock>
    <react-native-mock>
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            SETTINGS
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
  </react-native-mock>
  <react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
    <react-native-mock>
      <react-native-mock />
      <react-native-mock>
        <react-native-mock>
          <react-native-mock>
            <react-native-mock>
              Quickly restore ubiquitous outside the box thinking and long-term high-impact leadership skills. Appropriately.
            </react-native-mock>
          </react-native-mock>
        </react-native-mock>
      </react-native-mock>
    </react-native-mock>
  </react-native-mock>
</react-native-mock>
`;
